contracts/cw20-dao-crowdfund/src/contract.rs

```diff
@@ -16,6 +16,18 @@ use crate::state::{State, FUNDING_TOKEN_ADDR, GOV_TOKEN_ADDR, STATE};
 const CONTRACT_NAME: &str = "crates.io:cw20-dao-crowdfund";
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+const MAINNET_CHAIN_ID: &str = "juno-1";
+const MAINNET_DAO_UP_ADDR: &str = "juno1puc0wp0dhzp9hyvp7tw7edvx496ekewsz0ahrmhtnfaq0tzr0ggs2t42tx";
+const MAINNET_PUBLIC_PAYMENT_DENOM: &str = "ujuno";
+const TESTNET_DAO_UP_ADDR: &str = "juno1pla9cky8drecdsqwp3uh6l76yhpkfkaeqz9xkn3xwk6vjlr2y22s255x3t";
+const TESTNET_PUBLIC_PAYMENT_DENOM: &str = "ujunox";
```

Comment:
What do you think about calling this `CHAIN_ID` and `DAO_UP_ADDR` and deploying different versions on testnet / mainnet?